eloquent/distribution.yaml

```diff
@@ -521,7 +521,7 @@ repositories:
       test_pull_requests: false
       type: git
       url: https://github.com/eProsima/Fast-RTPS.git
-      version: 141b778c66c4f1f7aeb5c55551c173e3237beae9
+      version: 1.9.x
```

Comment:
For other rosdistros we've specified the precisely released Fast-RTPS tag. That was due in part to not having a stable development branch to track but I could still see merit in keeping an exact release for upstream code. Only CycloneDDS and Fast-RTPS are really in this situation.

Reply:
I switched this back to the SHA